phpstan-baseline.neon

```diff
@@ -1,5 +1,15 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Class MongoDB\\\\Laravel\\\\Query\\\\Grammar does not have a constructor and must be instantiated without any parameters\\.$#"
+			count: 1
+			path: src/Connection.php
+
+		-
+			message: "#^Class MongoDB\\\\Laravel\\\\Schema\\\\Grammar does not have a constructor and must be instantiated without any parameters\\.$#"
+			count: 1
+			path: src/Connection.php
+
```

Comment:
It tried using the `@phpstan-ignore new.noConstructor` in the code, but that triggers an other error `ignore.unmatchedIdentifier` on Laravel 12, where the constructor has this parameter.